build.sbt

```diff
@@ -1,6 +1,6 @@
 // Convenient setting that allows writing `set scalaVersion := dotty.value` in sbt shell to switch from Scala to Dotty
 val dotty = settingKey[String]("dotty version")
-dotty in ThisBuild := dottyLatestNightlyBuild.get
+dotty in ThisBuild := "0.2.0-RC1"
```

Comment:
You weren't actually using the latest nightly build previously, because the version of sbt-dotty used looked for the latest nightly in dotty_0.1 but we're now at dotty_0.2 (and soon at _0.3). I think it makes more sense to use a stable version now that we have one, and use the https://github.com/lampepfl/dotty-community-build to make sure the latest nightly still works.